modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -362,12 +362,27 @@ $ openshift-install version
 release image quay.io/openshift-release-dev/ocp-release:4.y.z-x86_64
 ----
 
-.. Locate all `CredentialsRequest` objects in this release image that target the cloud you are deploying on:
+.. Set a `$RELEASE_IMAGE` variable with the release image from your installation file by running the following command:
 +
 [source,terminal]
 ----
-$ oc adm release extract quay.io/openshift-release-dev/ocp-release:4.y.z-x86_64 --credentials-requests --cloud=azure
+$ RELEASE_IMAGE=$(./openshift-install version | awk '/release image/ {print $3}')
 ----
+
+.. Extract the list of `CredentialsRequest` custom resources (CRs) from the {product-title} release image by running the following command:
++
+[source,terminal]
+----
+$ oc adm release extract \
+  --credentials-requests \
+  --to=<path_to_directory_for_credentials_requests> \// <1>
+  --from=$RELEASE_IMAGE \
+  --included \// <2>
+  --install-config=<path_to_directory_with_installation_configuration>/install-config.yaml <3>
```

Comment:
nit: over in `modules/cco-ccoctl-deleting-sts-resources.adoc`, you currently suggest 

```asciidoc
  --included // <2>
```

but here you don't have the `//` bit.  I'm not clear on what the `//` is doing, which is fine, but  I'm surprised that in what seems like a similar situation, you sometimes use it, and sometimes don't.

Reply:
In `cco-ccoctl-deleting-sts-resources.adoc`, that is a clue that I used to have a line after ` --included`; as that line currently stands it's not needed. I'll rearrange as you suggest elsewhere and tidy up :nerd_face: 

As for what is going on in the asciidoc, this block you've highlighted is actually a good example of the issue it's aimed at fixing:
```
[1]  --to=<path_to_directory_for_credentials_requests> \// <1>
[2]  --from=$RELEASE_IMAGE \
[3]  --included \// <2>
[4]  --install-config=<path_to_directory_with_installation_configuration>/install-config.yaml <3>
```
[1] This line has a callout and more lines in the command after it. The callout breaks the text highlighting for the rest of the code block unless you put `// ` in front of it. So, we have line break (`\`), don't mess up the highlighting (`// `), and make a callout (`<1>`).
[2] no callout, no need to worry about highlighting
[3] same as [1]
[4] end of block, no subsequent lines means we don't need to worry about the highlight breaking